cmd/containerd-shim/main_unix.go

```diff
@@ -75,7 +75,7 @@ func init() {
 	flag.StringVar(&addressFlag, "address", "", "grpc address back to main containerd")
 	flag.StringVar(&workdirFlag, "workdir", "", "path used to storge large temporary data")
 	flag.StringVar(&runtimeRootFlag, "runtime-root", process.RuncRoot, "root directory for the runtime")
-	flag.StringVar(&criuFlag, "criu", "", "path to criu binary")
+	flag.StringVar(&criuFlag, "criu", "", "path to criu binary (deprecated: do not use)")
```

Comment:
The entire shim v1 itself is already deprecated, so maybe this message isn't necessary

Reply:
Yeah, I doubt many people will use it manually from the command-line, but when running the command, it's not printed that it's deprecated, and I think it's useful to at least make it visible that the option itself should not be used?

Reply:
The v1 shim is deprecated but working. The flag is deprecated and it doesn't work (ignored). So I'd like to keep the message.